proposal.rst

```diff
@@ -96,53 +95,61 @@ Background Information
 similar programmes before?  If so, tell us about how it went, and any areas you
 would have liked more help with.**
 
-FILLME
+Unfortunately, no.
 
 **Please tell us about any previous experience you have with Xapian, or other
 systems for indexed text search.**
 
-FILLME
+I have good c++ programming skills and a few knowledge about Lucene/Solr. I have used Solr to provide several simple restful APIs about searching stock data. Besides, I have joined a startup search engine company, DingFu Data, as an intern engineer. My works mainly on filtering massive of invalid data before importing documents into the search engine, and design corresponding data schema for web service.
 
 **Tell us about any previous experience with Free Software and Open Source
 other than Xapian.**
 
-FILLME
+I have no previous experience.
 
 **What other relevant prior experience do you have (courses taken at college,
 hobbies, holiday jobs, etc)?**
 
-FILLME
+1. Distributed System
+2. C++ Programming
+3. Advanced Data Structure
+4. Introduction to System
+5. Database System Concept
+6. Computer Software Design and Implementation
 
 **What development platforms, tools and methods do you prefer to use?**
 
-FILLME
+I develop softwares in the ubuntu 14.04 platform. Mostly, I use VS Code or Sublime for writing codes. In some cases without a graphical interface, Vim is my primary choice.
+
 
 **Have you previously worked on a project of a similar scope?  If so, tell us
 about it.**
 
-FILLME
+I have worked on a financial search engine project. In this project, user can search stock data, financial report and financial news from website. 
+My duty is that filter massive of invalid data before importing documents into the search engine, and design corresponding data schema for web service.
 
 **What timezone will you be in during the coding period?**
 
-FILLME
+UTC+8
 
 **Will your Summer of Code project be the main focus of your time during the
 program?**
 
-FILLME
+Yes, this project will be treated as the primary one except some unresistible factors.
```

Comment:
What are these "unresistible factors"?

It's not necessarily a problem if there are some small existing commitments, but we really need to be clear on what they are.

Reply:
I have the lumbar disc herniation (LDH). Sometimes, I have to go to my doctor. This one is the main unresistible factors.